.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: go
 go:
   - 1.5.1
-branches:
-  only:
-    - master
```

Comment:
is it necessary to remove this? It's located in other repos (like [deis/builder](https://github.com/deis/builder/blob/master/.travis.yml#L2-L4)) and the script still seems to run...